timereport/chalicelib/lib/slack.py

```diff
@@ -24,13 +24,12 @@ def slack_payload_extractor(req):
     :param req: The request data from slack
     :return: dict
     """
-
     # parse_qs makes a list of all values, and that's why the v.pop() is necessary
     return {k: v.pop() for (k,v) in parse_qs(req['body']).items()}
 
 
 def verify_token(token):
-    if token == os.environ['SLACK_TOKEN']:
+    if token == os.getenv('SLACK_TOKEN', 'fake_token'):
```

Comment:
Why do we specify this default "fake token"?

Reply:
first and foremost we should always use os.getenv for variables instead of environ since we will Errors if a OS environ is not set, the fake_token is defaulted to because we use a fake_token when we test this locally and it's cumbersome to set a token value for testing locally everytime so we can default to it for dev and have it set to actual values in prod with env variables.

thoughts?

Reply:
If we test locally then we should have a config that is used containing a token like that. I think the app should provide an error if no token is found.

Reply:
ok removed